src/bpf/bpfd-manager.h

```diff
@@ -0,0 +1,24 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+#pragma once
+
+#include "hashmap.h"
+#include "sd-bus.h"
+#include "sd-event.h"
+
+#include "macro.h"
+
+/* Polling interval for BPF timer  */
+#define BPF_TIMER_INTERVAL_USEC 5 * USEC_PER_SEC /* 5.0 seconds */
```

Comment:
Should we make this configurable?

Reply:
I've seen other services define their timer intervals in *-manager.h like this, so I think this is consistent with the style. I will increase the interval though since I forgot - 5 seconds is too short.